### ISSUE TITLE
[DOC] - Missing permissions on self hosted bot?

### ISSUE THREAD

**Issue by bradtaniguchi** [BeccaLyria/discord-documentation#559]
### What is incorrect in the documentation?

I'm not 100% sure if its permission related, but I **think** it is. As my bot doesn't have any permission set except 1:
![image](https://user-images.githubusercontent.com/10079147/135766706-37395093-cafc-46d4-a2e8-2eab03bcc212.png)

Otherwise, when I run the bot, I get an error with the slash commands:
```
debug: 2021-10-03 18:14:59: registering to home guild only
error: 2021-10-03 18:14:59: There was an error in the slash command register:
error: 2021-10-03 18:14:59: {"errorMessage":"Missing Access","errorStack":"DiscordAPIError[50001]: Missing Access\n    at SequentialHandler.runRequest (/workspaces/discord-bot/node_modules/@discordjs/rest/dist/lib/handlers/SequentialHandler.js:198:23)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async SequentialHandler.queueRequest (/workspaces/discord-bot/node_modules/@discordjs/rest/dist/lib/handlers/SequentialHandler.js:99:20)"}
error: 2021-10-03 18:14:59: failed to register commands.
```

### What should that text say?

Should there be a section in the docs specifying what permissions to set up the bot with at all? Or maybe I just missed it, or my issue when running the bot is totally unrelated?

**Comment by nhcarrigan** [comment]
The [Deploy Code](https://github.com/BeccaLyria/discord-documentation/blob/main/docs/deploy-code.md) steps should be modified to remove the invite example and document how to generate an invite link from the dashboard. Additionally, the need for the `application.commands` scope (along with the present `bot` scope) should be noted there as well.

The [Set Permissions](https://github.com/BeccaLyria/discord-documentation/blob/main/docs/set-permissions.md) step notes all of the permissions the bot requires, and should be linked in the deploy code step for reference.

**Comment by nhcarrigan** [comment]
@bradtaniguchi for your reference, these are the settings needed for generating the invite URL:

![image](https://user-images.githubusercontent.com/63889819/135766885-974103aa-df89-447a-9c4f-5c27b52dc9a9.png)

**Comment by princeprag** [comment]
@nhcarrigan  is it open for grab?

**Comment by nhcarrigan** [comment]
Go for it!

**Comment by bradtaniguchi** [comment]
On the topic of the error stack I got above, my issue was tied with the permissions I invited the bot with. Here's more or less the steps/mistakes I made:

1. I forgot to invite the bot into the guild completely, hence it couldn't register the commands.
2. I copied the default URL from the official discord docs, which are missing the required properties
3. I kicked the bot, and re-invited it using the URL generated from the above settings posted above.
4. Accepted the following permissions discord showed to me when I invited the bot:
![image](https://user-images.githubusercontent.com/10079147/135948469-b70fbe25-fdf0-4c9e-ae20-d23d10742ad1.png)
5. Bot go through the above stack trace and exploded due to something else (database issue), but this is unrelated to any of the above settings 👍
